outlines/blaze.md

```diff
@@ -0,0 +1,56 @@
+# Blaze
+
+1. Introduction to Blaze -- the tracker-backed handlebars like templating syntax
```

Comment:
Technically, the syntax is called spacebars, and the runtime is blaze. Like jsx and react.